components/rms/slurm/SPECS/slurm.spec

```diff
@@ -152,6 +152,10 @@ BuildRequires: pkgconfig
 %endif
 %endif
 
+%if 0%{?openEuler}
+BuildRequires: perl perl-devel
```

Comment:
perl-devel already requires perl, so perl is also unnecessary

Reply:
Removed it!